samples/benchmarks/coremark/boards/nrf5340dk_nrf5340_cpuapp.conf

```diff
@@ -0,0 +1 @@
+CONFIG_BOARD_ENABLE_CPUNET=y
```

Comment:
`APP_NRF53_CPUNET_RUN` should select it I think

Reply:
It doesn't work as APP_NRF53_CPUNET_RUN  is sysbuild option and not APP_CORE option